bzip2/PSPBUILD

```diff
@@ -9,30 +9,24 @@ license=('custom')
 depends=()
 makedepends=()
 optdepends=()
-source=(
-    "ftp://sourceware.org/pub/${pkgname}/${pkgname}-${pkgver}.tar.gz"
-    "${pkgname}-${pkgver}-PSP.patch"
-)
-sha256sums=(
-    'a2848f34fcd5d6cf47def00461fcb528a0484d8edef8208d6d2e2909dc61d9cd'
-    'SKIP'
-)
-
-prepare() {
-    cd "$pkgname-$pkgver"
-    patch -p1 < ../${pkgname}-${pkgver}-PSP.patch
-}
+source=("git+https://gitlab.com/fjtrujy/bzip2.git#commit=859485c930a678df80df4bbaf5aab22889d76683")
```

Comment:
The upstream tar.gz for this version and a patch file would be preferred here.

Reply:
I have created a PR to the original repo, it is waiting to be merged. Let's keep it like this, and I will update it once, it is merged

Reply:
That makes sense, thanks for the info